user_resource_mapping.go

```diff
@@ -149,6 +149,13 @@ func (m *UserResourceMapping) ownerPerms() ([]Permission, error) {
 		return OwnerPermissions(m.ResourceID), nil
 	}
 
+	if m.ResourceType == InstanceResourceType {
+		return []Permission{
```

Comment:
I wonder if we should just `return OperPermissions()` here, similar to `return OwnerPermissions()` above?

Reply:
OperPermissions is used in a lot of places and is actually special cased _not_ to return the instance owner permissions because it is used for tokens and not for session auth. So this was the easy solution to only have session auth enabled as instance owners.

Alternatively, we can allow tokens to be instance owners but if we did that through operator tokens that would be a bit of a deviation from what that currently means since that is still scoped to an org.